ffmpeg/build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+link="https://ffmpeg.org/releases/ffmpeg-5.1.2.tar.bz2"
```

Comment:
Please add a cleanup step before the download step.
See an example on [nginx](https://github.com/unikraft/static-pie-apps/blob/master/nginx/build.sh#L3).